src/GSoft.Extensions.MediatR.Analyzers.Tests/GSoft.Extensions.MediatR.Analyzers.Tests.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Each Roslyn analyzer is tested for successful scenarios and failure scenarios:

![image](https://user-images.githubusercontent.com/14242083/221236136-494a4e27-6cec-4a3e-bae6-9bae9671ccfd.png)